src/nvim/testdir/test_usercommands.vim

```diff
@@ -88,6 +179,8 @@ func Test_Ambiguous()
   Do
   call assert_equal('also', g:didthat)
   delcommand Dothat
+
+  call assert_fails("\x4ei\041", ' you demand a ')
```

Comment:
This calls `:Ni!`, which displays "Do you demand a shrubbery?" in Vim, but I could not find where/how.

Should that be replaced with an internal Neovim command then?

Reply:
"internal" how? maybe `:checkhealth` ?

Reply:
Sounds good.
Any idea where the `:Ni!` is coming from - did not find anything looking for e.g. shrubbery in the sources.

Reply:
we removed `ex_ni`. This was not intended to remove a cute easter egg; `ex_ni` was for "not implemented" commands, but Neovim has no "not implemented" stuff because Neovim has the same features in all builds/platforms.

Reply:
I see.
I am still confused by not finding "shrubbery" in Vim's sources though.

Reply:
> I am still confused by not finding "shrubbery" in Vim's sources though.

￼Mystery solved thanks to osse on `#vim`: 
https://github.com/vim/vim/blob/12ee7ff00b91d852e060bb24951d1c94239863eb/src/ex_docmd.c#L1909-L1914
https://github.com/vim/vim/blob/12ee7ff00b91d852e060bb24951d1c94239863eb/src/usercmd.c#L547-L560

Reply:
Yeah. That's pretty silly though. If we can adjust the test to use `:checkhealth` then that would be preferred. Otherwise  restore the easter egg.

Reply:
Yes, we should not have it.

I guess this test was added to check that code path, and it is fine to adjust the test for it to match what Neovim does (although we likely do not need it for coverage reasons).  Good to have the test (almost) in sync though.